packages/shared/lib/services/notification/webhook.service.ts

```diff
@@ -1,3 +1,5 @@
+// @packages: jobs, server
```

Comment:
Thoughts here? I sometimes have trouble remembering which package context a shared service is used.

Reply:
mmh is this automatic? I don't think it will be maintained correctly over the time.
imo not everything has to in shared, with a little bit more split we could find imports more quickly

Reply:
I am also worried about this becoming very quickly out-of-date and potentially misleading.

Reply:
Removed with https://github.com/NangoHQ/nango/pull/1644/commits/34d57f2bc62326fc4a5c08c8d14173cf114bd8f4